libs/data-access/advert/src/lib/constants.ts

```diff
@@ -0,0 +1,14 @@
+export enum ActionTypes {
+  CREATE = 'ADVERT/CREATE',
```

Comment:
all action constants should be unique, CREATE -> ADVERT_CREATE
same below and action names also should be unique, because if use different action in same component you get conflicts.

Reply:
I was following your example: 
```
  LOG_IN = 'authorizations/LOG_IN',
  LOG_IN_SUCCESS = 'authorizations/LOG_IN_SUCCESS',
  LOG_IN_FAILED = 'authorizations/LOG_IN_FAILED',
```
"AUTHORIZATIONS_LOG_IN" or "LOG_IN"?